modules/external-api/src/main/java/org/opencastproject/external/common/ApiMediaType.java

```diff
@@ -54,6 +54,8 @@ private static ApiFormat extractFormat(String mediaType) {
     final String format;
     if (subtype.contains("+")) {
       format = subtype.substring(subtype.indexOf("+") + 1);
+    } else if ("*".equals(subtype)) {
```

Comment:
What is the reasoning behind this?

Reply:
See the comment above

Reply:
hi @staubesv, I just thought about it again. Maybe it is better to just set JSON here and don't use UNSPECIFIED. Or even removing the whole ApiFormat enum since we only support JSON. What do you think?

Reply:
changed to fall back to JSON